setup.sh

```diff
@@ -1,11 +1,33 @@
-#!bin/bash
+#!/bin/bash
 
 setxkbmap -option "caps:swapescape"
 sudo add-apt-repository ppa:mmstick76/alacritty
 sudo apt update
-sudo apt install libpq-dev postgresql postgresql-contrib fonts-firacode libx11-dev ripgrep curl alacritty xclip vim-gtk3 git libinput-tools xdotool nodejs build-essential ruby -y
-curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+sudo apt install -y \
+  alacritty \
+  build-essential \
+  curl \
+  fonts-firacode \
+  git \
+  libinput-tools \
+  libpq-dev \
+  libx11-dev \
+  nodejs \
+  npm \
+  postgresql \
+  postgresql-contrib \
+  ripgrep \
+  ruby \
+  vim-gtk3 \
+  xclip \
+  xdotool
+
+npm i -g n && \
```

Comment:
I removed the nodesource Node install, since you were already installing from apt repos. Also added this, since the apt repo version would be old.